commit 8ceadb812f710781a40549f01a6c4b9baab6001e
Author: Ben Skeggs <bskeggs@redhat.com>
Date:   Fri Mar 17 00:48:29 2017 +0000

    drm/nouveau/disp/gp102: fix cursor/overlay immediate channel indices
    
    [ Upstream commit e50fcff15fe120ef2103a9e18af6644235c2b14d ]
    
    Signed-off-by: Ben Skeggs <bskeggs@redhat.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1440_8ceadb812f710781a40549f01a6c4b9baab6001e.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1440_8ceadb812f710781a40549f01a6c4b9baab6001e.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
